README.md

```diff
@@ -80,14 +80,20 @@ require "stripe"
 
 Stripe::Charge.list(
   {},
-  :api_key => "sk_test_...",
-  :stripe_account => "acct_..."
+  {
+    :api_key => "sk_test_...",
+    :stripe_account => "acct_...",
+    :stripe_version => "2018-02-28"
+  }
 )
 
 Stripe::Charge.retrieve(
   "ch_18atAXCdGbJFKhCuBAa4532Z",
-  :api_key => "sk_test_...",
-  :stripe_account => "acct_..."
+  {
```

Comment:
so the issue is that the Charge Retrieve API (And any retrieve) supports `expand`. And that would be in the first hash. So really the call should  be 

```
Stripe::Charge.retrieve(
  "ch_18atAXCdGbJFKhCuBAa4532Z",
  {}, // where expand would go
  {
    :api_key => "sk_test_...",
    :stripe_account => "acct_...",
    :stripe_version => "2018-02-28"
  }
)
```

right?

Reply:
Okay, it's _really_ bad that this is confusing to even us, but the correct invocation with an expansion is this:

``` ruby
Stripe::Charge.retrieve( {id: "ch_1COBry2eZvKYlo2CmgIUoaBZ", expand: [:customer] }, { stripe_version: "2018-02-28" })
```

The method won't take three parameters:

``` ruby
[28] pry(main)> Stripe::Charge.retrieve("ch_1COBry2eZvKYlo2CmgIUoaBZ", {}, {})
ArgumentError: wrong number of arguments (3 for 1..2)
from /Users/brandur/stripe/stripe-ruby/lib/stripe/api_resource.rb:61:in `retrieve'
```

Reply:
Ah damn it you're right I though it depended on the language but it does not.